package/MDAnalysis/analysis/bat.py

```diff
@@ -31,11 +31,11 @@
 
 This module contains classes for interconverting between Cartesian and an
 internal coordinate system, Bond-Angle-Torsion (BAT) coordinates
-:cite:p:`Chang2003`, for a given set of atoms or residues. This coordinate
+:footcite:p:`Chang2003`, for a given set of atoms or residues. This coordinate
```

Comment:
Do we document how we do citations anywhere in our docs? We probably should tell folks that cite -> footcite now

Reply:
Are there any peculiarities to footbibliography that should be stated somewhere? (I don't know how/if it differs from the old approach)

Reply:
Ticked :)

I don't think so, no more than `bibliography` is. The footbib is used for local bibliographies (on the same page as the reference), so we can remove stuff like the `a-` and `b-` prefixes we had before. The normal bibliography is for global bibliographies, so one reference list for the entire repo. This is all readily available on the sphinx bibtex extension docs though. I was planning to add in some notes in the mdanalysis-sphinx-theme -- https://github.com/MDAnalysis/mdanalysis-sphinx-theme/issues/26

The one catch is that there's meant to be one footbib per reference per page, so if footbibliographies are created multiple times for the same reference, then only the first one will ahve the reference and all links will go there. Again I got this from the official docs.

Reply:
Could you maybe raise an issue in the userguide for docs on this? This is the kind of info any contributor new or old should be told to look at so imho, even if it's just a reference to the theme docs, your above explanation should exist there in some way.

Reply:
Sure, done